docs/modules/ROOT/pages/tests.adoc

```diff
@@ -152,6 +167,18 @@ To navigate from your Testspace Project down to an individual test case, select
 *** Test sequence (contains tests) Example: `e2e_tests.langchain.test_compatibility_rag`
 **** Test cases (passed, failed, skipped, etc.) Example: `Test rag: openai embedding | openai llm | cassandra | rag custom chain`
 
+=== LangSmith trace
+
+Within individual test cases, https://smith.langchain.com/[LangSmith]{external-link-icon} traces are also available to view.
+
+A LangSmith trace displays the test's entire LLM chain, including the input prompt, the generated response, token spend, and the metadata associated with the response.
```

Comment:
I would add a note that the link only appears if you are logged in into testspace 

I already see a salesperson following this guide and getting errors without knowing how to go further in the dashboard